modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/uuid/ItUuidDmlTest.java

```diff
@@ -17,16 +17,78 @@
 
 package org.apache.ignite.internal.sql.engine.datatypes.uuid;
 
+import static org.apache.ignite.lang.IgniteStringFormatter.format;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 import java.util.UUID;
 import org.apache.ignite.internal.sql.engine.datatypes.DataTypeTestSpecs;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.BaseDmlDataTypeTest;
 import org.apache.ignite.internal.sql.engine.datatypes.tests.DataTypeTestSpec;
+import org.apache.ignite.internal.sql.engine.datatypes.tests.TestTypeArguments;
 import org.apache.ignite.internal.sql.engine.type.UuidType;
+import org.apache.ignite.lang.IgniteException;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
 /**
  * Tests for DML statements for {@link UuidType UUID data type}.
  */
 public class ItUuidDmlTest extends BaseDmlDataTypeTest<UUID> {
+    /** {@code INSERT} with dynamic parameters is not allowed for compatible types. */
+    @ParameterizedTest
+    @MethodSource("convertedFrom")
+    public void testInsertFromDynamicParameterFromConvertible(TestTypeArguments arguments) {
+        var t = assertThrows(IgniteException.class, () -> {
```

Comment:
please use `org.apache.ignite.internal.sql.engine.util.SqlTestUtils#assertThrowsSqlException`

Reply:
like this
`
        assertThrowsSqlException(
                Sql.STMT_VALIDATION_ERR,
                "Values passed to VALUES operator must have compatible types",
                () -> runSql("INSERT INTO t VALUES (1, ?)", arguments.argValue(0))
        );
`

Reply:
@ygerzhedovich fixed but this activity beyond the scope of this issue.